src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -467,11 +467,19 @@ ExportQueryExecutionTrees::idToStringAndTypeForEncodedValue(Id id) {
         }
         double dIntPart;
         // If the fractional part is zero, write number with one decimal place
-        // to make it distinct from integers. Otherwise, use `%g`, which uses
+        // to make it distinct from integers. Otherwise, use `%.13g`, which uses
```

Comment:
Is this just a hack or a standard way of exporting floating-point numbers in SPARQL?